src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java

```diff
@@ -129,9 +133,9 @@ public void revokeAccessTokens(String id, String token) {
             LOGGER.warn(String.format("Unable to revoke access token on auth-server %s: %s",
                     authServerHome, e.getMessage()));
             throw e;
-        } catch(OsiamRequestException e) {
+        } catch (UnauthorizedException e) {
             LOGGER.warn("Error revoking all access tokens", e);
-            throw e;
+            throw new InvalidTokenException("The provided access token is not valid", e);
```

Comment:
We don't log the exception in the other log statements. I also don't think it's a good idea, it just clutters the logs for operations, we should just log the message:
`LOGGER.warn("Error revoking all access tokens: " + e.getMessage())`